create_infotexts.py

```diff
@@ -137,15 +245,13 @@ def generate_infobox_template(item, img, places_mapping):
     infobox += "| title               =\n"
 
     infobox += "| description        = {{sv| "
+
     if not item["Beskrivning"] == "":
-        # print("item['Beskrivning']: {}".format(item["Beskrivning"]))
-        cleaned_beskrivning = re.sub(" Svenska Cypernexpeditionen\.?", "", item["Beskrivning"])
-        infobox += cleaned_beskrivning
-        if not cleaned_beskrivning.endswith("."):
-            infobox += "."
-    infobox += " Svenska Cypernexpeditionen 1927-1931."
-    if not item["Nyckelord"] == "" and not item["Nyckelord"] == "Svenska Cypernexpeditionen":
-        infobox += "<br /> ''Nyckelord:''\n" + item["Nyckelord"]
+        infobox += enrich_description(item)
```

Comment:
Yes, that's the crux. Had we not found the issue with filenames not being good enough when created I would have implemented this cleanre and simpler. That's a very strong argument for a stricter separation and emphasis on data description and data quality assurance at the beginning of next batchupload.